catalog/app/containers/Package/index.js

```diff
@@ -224,20 +225,49 @@ Install.propTypes = {
   owner: PropTypes.string.isRequired,
 };
 
+const sizes = [
```

Comment:
This feels overly complicated/verbose... Just do this? https://stackoverflow.com/questions/15900485/correct-way-to-convert-size-in-bytes-to-kb-mb-gb-in-javascript?answertab=active#tab-top

(Replace 1024 with 1000 or whatever.)

Reply:
apparently `**` is faster than `Math.pow` (linter doesn't like `Math.pow` either). i think my solution should be the SO answer lol. seemed clean to me :) what we actually need are unit tests and @nl0 is working on those. unless this code is really bad would prefer to leave as is.

Reply:
anyway, `**` is not really the issue. if you prefer i can take the log and jump right to the correct units if stepping through the array is hard to read.

Reply:
I didn't even mean `**` vs `pow` or `log`. I meant, the `lim: 3`, `lim: 6`, etc. are unnecessary - you can just use an array?

Reply:
yeah, done